MIP/MIP-60/README.md

```diff
@@ -0,0 +1,228 @@
+3. The relayer must cover the cost of submitting transactions on the target chain. If the relayer's funds are low (or gas price is high), the relayer may not be able to submit the transaction.
+
+### Designs
+
+**HTLC-based design**
+
+Our initial bridge design, [RFC-40](https://github.com/movementlabsxyz/rfcs/blob/main/0040-atomic-bridge/rfc-0040-atomic-bridge.md), is based on an _atomic swap_ (4. above).  The current architecture of our bridge, [MIP-39](https://github.com/movementlabsxyz/MIP/blob/mip-move-bridge-architecture/MIP/mip-39/README.md), borrows ideas from _lock/mint_ and _swap_ ([HTLC](https://en.bitcoin.it/wiki/Hash_Time_Locked_Contracts)):
+ 
+- if a user wants to bridge from Ethereum to Movement Network, they have to submit two independent transactions: one on Ethereum (lock) and one on Movement Network (mint). This is not user friendly and may be a barrier to adoption.
+<!-- A bridge is slightly different to a swap in the sense that the two transactions (lock and mint) are not meant to be decoupled.-->
+- another issue is that the first transfer of MOVE tokens requires a user to have some _gas tokens_ (the token used to pay execution fees on the Movement Network). If the gas token is the `$L2MOVE` token, we have to implement a mechanism to sponsor the first transfer of MOVE tokens.
+
+**Trusted-Relayer design**
+
+> [!TIP]
+> A simple _lock/mint_, _burn/mint_ or _lock/unlock_ mechanism requires only one transaction from the user and is probably more user friendly.
+
+A simple design that is adopted by many L2 chains is the _lock/mint_ design. A user initiates a transfer on the source chain and the bridge operator is responsible for completing the transfer on the target chain. In this design, the trust assumptions are that the contracts on the source chain and the target chain are _correct_ and implement the lock and mint operations correctly (respectively burn and unlock for the opposite direction).
+
+[MIP-58](https://github.com/movementlabsxyz/MIP/pull/58/files) is an instance of a _lock/mint_ bridge. Once initiated on the L1 (source chain), the user does not need to interact with the L2 (target chain) to complete the transfer. The bridge operator is responsible for bundling/coupling the lock/mint transactions.  (Similarly for the opposite direction, the user only needs to interact with the L2 chain and the bridge operator is responsible for the coupling of the burn/unlock transactions.)
+
+## Security
+
+### Security - HTLC-based design
+
+See [MIP-39](https://github.com/movementlabsxyz/MIP/blob/mip-move-bridge-architecture/MIP/mip-39/README.md) for a complete discussion of the security of this type of design.
+
+### Security - Trusted-Relayer design
+
+See [Trusted Relayer design](#trusted-relayer-design).
+
+Assuming we have a _correct_ implementation of the lock/mint contracts on the source and target chains, and a trusted relayer, we may still have to deal with the following issues:
```

Comment:
These requirements are too specific to the design solution and they should be transferred to MIP-58..